src/main/java/com/sprint/mission/discodeit/async/binary_content/BinaryContentStorageAsyncService.java

```diff
@@ -0,0 +1,76 @@
+package com.sprint.mission.discodeit.async.binary_content;
```

Comment:
```suggestion
package com.sprint.mission.discodeit.async.binarycontent;
```

https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

패키지 이름은 어쩔 수 없이 인터넷 도메인이 숫자로 시작한다던가 특수문자가 있다던가 의 경우가 아닌 이상
전부 소문자에 공백 없이 짓습니다.